app/templates/components/forms/orders/order-form.hbs

```diff
@@ -143,11 +143,11 @@
                 <label for="payment_mode"><img src="/images/payment-logos/omise.png" alt="omise"></label>
               </div>
             {{/if}}
-            {{#if data.event.canPayByAliPay}}
+            {{#if data.event.canPayByStripe}}
               <div class="field">
                 {{ui-radio name='payment_mode' label=(t 'AliPay') value='alipay' current=data.paymentMode
                            onChange=(action (mut data.paymentMode))}}
-                <label for="payment_mode"><img src="/images/payment-logos/alipay.png" alt="alipay"></label>
+                <label for="payment_mode"><img src="/images/payment-logos/alipay.png" height="25" widht="100" alt="alipay"></label>
```

Comment:
Use semantic UI image classes, don't use arbitrary attributes, see how it is done for PayPal.

Reply:
@CosmicCoder96 Updated.